src/main/service/FeatureToggleService.ts

```diff
@@ -9,7 +9,8 @@ export class FeatureToggleService{
   }
 
   public static async getChallengedAccessFeatureToggle(): Promise<void> {
+    const is_development = String(config.get('is_dev'));
     const challengedAccessFt = String(config.get('featureToggles.ft_challengeAccess'));
-    return await LaunchDarklyClient.instance.variation(challengedAccessFt, false);
+    return await LaunchDarklyClient.instance.variation(challengedAccessFt, is_development? true : false);
```

Comment:
You know that you don't need to do `is_development` check? You can simply pass `is_development`. Also, it doesn't follow naming convention.

Reply:
it has been changed